src/tutorials/dropbox.md

```diff
@@ -0,0 +1,72 @@
+# Setting up the Dropbox daemon
+
+- Time needed: ~10 minutes
+
+Dropbox has a very useful [Linux version](https://www.dropbox.com/install-linux) that can be installed into your Platform.sh application with a bit of intermediate [build hook](/configuration/app-containers.html#hooks) hacking.
+
+## Install the daemon
+
+Per the instructions on [installing the Linux client](https://www.dropbox.com/install-linux), you can install the client via a simple one-liner shell command, but I needed to modify it slightly so modify your build hook to add this to it -
+
+```yaml
+hooks:
+  build: |
+    wget "https://www.dropbox.com/download?plat=lnx.x86_64" -O archive.tar
+    tar xzvf archive.tar && rm archive.tar
+```
+
+
+
+The build hook runs when your application's codebase is being assembled for deployment, so the filesystem is still writable.  This means that the above command will create a `~/.dropbox-dist` directory in the home directory of your application, aka "the application root".  Starting the daemon is _almost_ as simple as running `~/.dropbox-dist/dropboxd` when you `ssh` into your application's container.
+
+## Creating the necessary writable directories
+
+Before we can start the daemon, we'll need to add a couple directories that Dropbox expects to be there and to be writable.  The main one is `~/Dropbox`, which contains **your entire Dropbox**.  (There may be a way to only mount part of it, and please do open a Merge Request to these docs if you figure it out).  For obvious reasons, this directory needs to be at least as large as all the contents of your Dropbox or you're going to have a Bad Time.
+
+The other is a metadata/etc directory at `~/.dropbox`.  This author's `.dropbox` directory is 43M, so it doesn't appear to take up too much room.  
+
+The combined total of these two directories needs to be smaller than whatever you've specified in the [`disk` parameter](/configuration/app-containers.html#disk) in your `.platform.app.yaml`, or put another way - the `disk` parameter needs to be larger than your total Dropbox usage.  (Don't forget to add a nice buffer on top of that too, because if your writable disk space fills up completely then Bad Things can happen).
```

Comment:
This can also be greatly condensed and formalized.  Suggested: 

"Because Dropbox files are mirrored on your writable disk mount, your application container's `disk` size needs to be large enough to accommodate all of your Dropbox files as well as any user-uploaded files and your own code.  You should also include sufficient extra space to allow for growth over time.